.changeset/late-shoes-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-site': minor
+'@microsoft/atlas-css': minor
+---
+
+Adds .gradient-card Component, which applies a gradient border, border-width and border-radius to a container.
```

Comment:
_Nit:_ no need to capitalize `component`.

Reply:
Updated